src/removeTasks.js

```diff
@@ -0,0 +1,4 @@
+export default function removeTasks(event) {
+  event.stopPropagation();
+  event.target.parentNode.remove();
+}
```

Comment:
- [ ] Please make sure when the user deletes a task, the change is then reflected in the local storage. 
- [ ] Kindly ensure that when a task is deleted, all remaining items' indexes are updated, so they represent the current list order and are unique. As per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should%20update%20all%20remaining%20items%27%20indexes%2C%20so%20they%20represent%20the%20current%20list%20order%20and%20are%20unique(i.e.%20if%20you%27re%20deleting%20the%20first%20task%20index%201%20from%20the%20list%2C%20the%20index%20of%20the%20next%20task(2)%20should%20set%20to%201)..) requirement.